rts/lra/coordinator-quarkus/pom.xml

```diff
@@ -155,7 +155,9 @@
       <profile>
         <id>quarkus-uber</id>
         <activation>
-          <activeByDefault>true</activeByDefault>
+          <property>
+            <name>!quarkus-uber</name>
```

Comment:
I don't understand this. The profile will run when you specify -Pquarkus-uber but when you do -Dquarkus-uber it will not run. What is the intent of this?

Reply:
I think it is the name of the property that's confusing. Perhaps it would be more intuitive to call it something like `no-quarkus-uber` instead.

Reply:
That's right. Thanks @xstefank and @mmusgrov I changed it to `no-quarkus-uber`.